frontend/packages/dev-console/integration-tests/features/gitops/add-Manifest-URL-ODC-3067.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Fetch applications using Manifest URL and create secret
+    User should be able to fetch the applications using Manifest URL and create github secret
+
+
+Background: 
+    Given user is at developer perspective
+
+
+@regression
+Scenario: No GitOps Manifest URLs found
+    Given user has installed GitOps Service Operator
+    When user clicks on GitOps nav item
+    Then user will see the message of No GitOps Manifest URLs found
+
+
+@regression, @manual
+Scenario: Add Manifest URL and No Application groups found message
+    Given user has installed the GitOps Service Operator
+    When user creates new namespace
+    And user goes to Search page
+    And user clicks on Resources dropdown
+    And user enters Namespaces in the Select Resource field
+    And user selects Namespace
+    And user clicks on newly created namespace
+    And user clicks on YAML tab
+    And user adds the Manifest URL under annotations
```

Comment:
We should mention which annotation the user needs to add, i.e `app.openshift.io/vcs-uri`

Reply:
Updated